MVVMIOS/ViewController.swift

```diff
@@ -22,9 +29,25 @@ class ViewController: UITableViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupActivityIndicator()
+        fetchDataFromViewModel()
+
+    }
+
+    func setupActivityIndicator() {
+
+    }
+
+    func fetchDataFromViewModel() {
         guard let url = URL(string: "https://google.com") else { return }
         viewModel.fetchFromNetwork(from: url)
+        viewModel.fetchCompleted = { _, errorText in
+            print(errorText)
+        }
```

Comment:
It's better to have as much logic outside the view controller as possible so it can be tested. Ideally this entire method implementation would just be `viewModel.fetchData()`. Let the view model be the place that knows which URL to hit, when to fetch from network, etc.

Reply:
Will remove the code related to determining the url from VC.